packages/@react-aria/landmark/stories/Landmark.stories.tsx

```diff
@@ -337,14 +343,14 @@ function IframeExample() {
       });
 
       setTimeout(() => {
-        document.body.removeAttribute('data-react-aria-top-layer');
+        document?.body.removeAttribute('data-react-aria-top-layer');
```

Comment:
ok it's going to be extremely annoying if we have to assert document is defined every time we use it 😂 

is there a tsconfig setting for that?

Reply:
will look, 😆

Reply:
don't we need this for SSR though?

Reply:
I hope no one is triggering landmark navigation during SSR. That seems like user error

Reply:
would it be better to use `!` then?
we wouldn't want to make this ok everywhere, there may be legitimate places where checking this is a good idea